components/accelerator/nvidia/query/xid/xid.go

```diff
@@ -948,6 +1078,20 @@ See below for guidelines on when to RMA GPUs based on excessive errors, via http
 		BusError:               false,
 		ThermalIssue:           false,
 		FBCorruption:           false,
+
+		// "Triggered when the GPU handles memory ECC errors on the GPU"
+		// "most instances can be resolved by simply resetting the GPU to retain optimal performance."
+		// ref. "Fire-Flyer AI-HPC: A Cost-Effective Software-Hardware Co-Design for Deep Learning" https://arxiv.org/abs/2408.14158
+		SuggestedActions: &common.SuggestedActions{
+			RepairActions: []common.RepairActionType{
+				common.RepairActionTypeCheckUserAppAndGPU,
+				common.RepairActionTypeRebootSystem,
```

Comment:
@photoszzt @xiang90 Same here.

Reply:
https://docs.nvidia.com/deploy/xid-errors/index.html#xid-error-listing xid 65 falls into hw error and driver error. Indeed needs reboot.